macros/generate_source.sql

```diff
@@ -15,26 +15,35 @@
 
 
 ---
-{% macro generate_source(schema_name, database_name=target.database, generate_columns=False, include_descriptions=False, table_pattern='%', exclude='') %}
+{% macro generate_source(schema_name, name = schema_name, database_name=target.database, generate_columns=False, include_descriptions=False, table_pattern='%', exclude='') %}
```

Comment:
To guarantee that it is a non-breaking change, we will need to move the new optional parameter to the end of the signature. Otherwise, it will break for anyone that is calling it like `generate_source("my_schema_name_here", target.database)`.

```suggestion
{% macro generate_source(schema_name, database_name=target.database, generate_columns=False, include_descriptions=False, table_pattern='%', exclude='', name=schema_name) %}
```